posthog/models/feature_flag/flag_matching.py

```diff
@@ -790,7 +790,8 @@ def get_all_feature_flags(
                     distinct_ids = [distinct_id, str(hash_key_override)]
                     query = """
                         WITH target_person_ids AS (
-                            SELECT team_id, person_id FROM posthog_persondistinctid WHERE team_id = %(team_id)s AND distinct_id IN %(distinct_ids)s
+                            SELECT team_id, person_id FROM posthog_persondistinctid WHERE team_id = %(team_id)s AND
+                            distinct_id = ANY(%(distinct_ids)s)
```

Comment:
this is in preparation for psycopg3 update?

Reply:
This *is* the psycopg3 update as with Django 4.2 it's used actively.

Reply:
hmm, oh I see, I was looking at docs for psycopg3.2.0, https://www.psycopg.org/psycopg3/docs/basic/from_pg2.html#server-side-binding - this is what I expect will cause the pg_sleep() tests to fail because it sends a SET command with parameter binding

but we can leave this be for when its actually released :P

Also had a look at the psycopg2 usage still in the app, we can remove it in a follow up indeed, no need to do it right now. 👍